.github/workflows/test.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.14, 1.15, 1.16, 1.17]
+        go-version: [1.15, 1.16, 1.17]
```

Comment:
Can you create a separate PR for this as I think there is a different audience that should have a look here.

Reply:
Okay.